FirebaseAuth/Sources/MultiFactor/TOTP/FIRTOTPSecret.m

```diff
@@ -81,9 +57,10 @@ - (void)openInOTPAppWithQRCodeURL:(NSString *)QRCodeURL {
   if ([[UIApplication sharedApplication] canOpenURL:url]) {
     [[UIApplication sharedApplication] openURL:url options:@{} completionHandler:nil];
   } else {
-    NSLog(@"Cannot open URL");
+    NSLog(@"URL cannot be opened");
```

Comment:
+1

Reply:
Per offline discussion, it's fine to just return here since this method doesn't return any error.

We'll see if we should make changes to this API later.